setup.cfg

```diff
@@ -76,6 +76,8 @@ invenio_base.blueprints =
     invenio_pages = invenio_pages.views:blueprint
 invenio_base.api_blueprints =
     invenio_pages_rest = invenio_pages.rest:blueprint
+invenio_base.functions =
```

Comment:
moderate: `functions` sounds a bit vague to me. It is not clear at what point will they be executed or with which purpose. Random ideas:
- init_functions
- post_app_functions
- with_app_functions